frontend/src/pages/Users.module.css

```
@@ -391,3 +397,181 @@
+
+.confirmInputWrap {
+  margin-top: var(--space-3);
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .confirmInputAnimate {
+    animation: slideDown 0.15s ease-out;
+  }
+}
+
+@keyframes slideDown {
```

Stylelint (17.10.0)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)